tickets_plus/cogs/override.py

```diff
@@ -0,0 +1,220 @@
+        It also sets the bot instance as a private attribute.
+        And finally initialises the superclass.
+
+        Args:
+            bot_instance: The bot instance.
+        """
+        self._bt = bot_instance
+        super().__init__()
+        logging.info("Loaded %s", self.__class__.__name__)
+
+    @app_commands.command(name="reload", description="Reloads the bot's cogs.")
+    @checks.is_owner_check()
+    @app_commands.describe(sync="Syncs the tree after reloading cogs.")
+    async def reload(self,
+                     ctx: discord.Interaction,
+                     sync: bool = False) -> None:
+        """Reloads the bot's cogs.
+
+        This command reloads all cogs in the EXTENSIONS list.
+        Reloads are atomic, so if one fails, it rolls back.
+        We can just import this submodule and iterate over the EXTENSIONS list.
+        You can also sync the tree after reloading cogs. Though this is not
+        to be used very often, as it has low rate limits.
+
+        Args:
+            ctx: The interaction context.
+            sync: Whether to sync the tree after reloading cogs.
+        """
+        await ctx.response.send_message("Reloading cogs...")
+        logging.info("Reloading cogs...")
+        for extension in cogs.EXTENSIONS:
+            await self._bt.reload_extension(extension)
+        await ctx.followup.send("Reloaded cogs.")
+        logging.info("Finished reloading cogs.")
+        if sync:
+            await self._bt.tree.sync()
+            logging.info("Finished syncing tree.")
+
+    @app_commands.command(name="close", description="Closes the bot.")
+    @checks.is_owner_check()
+    async def close(self, ctx: discord.Interaction) -> None:
+        """Closes the bot.
+
+        If used with a process manager, this will restart the bot.
+        If used without a process manager, this will close the bot.
+
+        Args:
+            ctx: The interaction context.
+        """
+        await ctx.response.send_message("Restarting...")
+        logging.info("Restarting...")
+        await self._bt.close()
+
+    @app_commands.command(
+        name="pull",
+        description="Pulls the latest changes from the git repo. DANGEROUS!")
+    @checks.is_owner_check()
+    async def pull(self, ctx: discord.Interaction) -> None:
+        """Pulls the latest changes from the git repo.
+
+        This command pulls the latest changes from the git repo.
+        This is a dangerous command, as it can break the bot.
+        If you are not sure what you are doing, don't use this command.
+
+        Args:
+            ctx: The interaction context.
+        """
+        await ctx.response.send_message("Pulling latest changes...")
+        logging.info("Pulling latest changes...")
+        pull = await asyncio.create_subprocess_shell(
+            "git pull",
+            stdout=asyncio.subprocess.PIPE,
+            stderr=asyncio.subprocess.PIPE,
+            cwd=statvars.PROG_DIR,
+        )
```

Comment:
Because of the nature of this command as documented, I would highly advise a confirmation prompt be required as a secondary step before command execution continues.

Reply:
Done